tests/cover/test_reflection.py

```diff
@@ -520,15 +520,15 @@ def test_copy_argspec_validates_function_name():
 
 class Container(object):
 
-    def funcy(self):
+    def function(self):
```

Comment:
Although `function` isn’t a builtin variable in Python, I think `funcy` is a better name here – there’s no risk of confusing it for an important keyword.